Nargo.toml

```diff
@@ -1,2 +1,3 @@
 [workspace]
-members = ["packages/merkle-trees"]
+compiler_version = ">=0.32.0"
```

Comment:
you already set a `compiler_version` in the ecdh package's `Nargo.toml`, no need to have this here
```suggestion
```